AdminWebsite/AdminWebsite.UnitTests/Controllers/PersonsControllerTest.cs

```diff
@@ -55,35 +56,183 @@ public void Setup()
         }
 
         [Test]
-        public async Task Should_return_request_if_match_to_search_term()
+        public async Task Should_return_searched_participants()
         {
-            _response.Add(new PersonResponse
-                            {
-                                Id = Guid.NewGuid(),
-                                ContactEmail = "jackman@hmcts.net1",
-                                FirstName = "Jack",
-                                LastName = "Mann",
-                                TelephoneNumber = "111222333",
-                                Title = "Mr",
-                                MiddleNames = "No",
-                                Username = "jackman@hmcts.net"
+            // Arrange
+            var additionalParticipantToReturn = new PersonResponse
+            {
+                Id = Guid.NewGuid(),
+                ContactEmail = "jackfilter@hmcts.net",
+                FirstName = "Filter",
+                LastName = "Participant",
+                TelephoneNumber = "111222333",
+                Title = "Mr",
+                MiddleNames = "No",
+                Username = "jackfilter@hmcts.net"
+            };
+            _response.Add(additionalParticipantToReturn);
+            
+            var expectedResponse = new List<PersonResponse>();
+            expectedResponse.Add(_response[0]);
+            expectedResponse.Add(additionalParticipantToReturn);
+            
+            _bookingsApiClient.Setup(x => x.PostPersonBySearchTermAsync(It.IsAny<SearchTermRequest>()))
+                .ReturnsAsync(_response);
+
+            _userAccountService.Setup(x => x.GetJudgeUsers()).ReturnsAsync(new List<JudgeResponse>());
+
+            _bookingsApiClient.Setup(x => x.PostJudiciaryPersonBySearchTermAsync(It.IsAny<SearchTermRequest>()))
+                .ReturnsAsync(new List<PersonResponse>());
+            
+            var searchTerm = "ado";
+
+            // Act
+            var result = await _controller.PostPersonBySearchTerm(searchTerm);
+
+            // Assert
+            var okObjectResult = result.Result.Should().BeAssignableTo<OkObjectResult>().Which;
+            okObjectResult.Value.Should().BeEquivalentTo(expectedResponse);
+            _bookingsApiClient.Verify(x => x.PostPersonBySearchTermAsync(It.Is<SearchTermRequest>(request => request.Term == searchTerm)), Times.Once);
```

Comment:
I was able to do this in nsubstitute and I think we can do this here too, it is however just a suggestion to consider. 

Can we add this query in line 79, so it would mean that we only return the the expected response for the given argument. Ultimately based on the response you get from that mocked client calls you can get the expected response so you don't have to do this added assertions. We can then do the same for the bookingsApiClient mock setup.